test-case/verify-kernel-boot-log.sh

```diff
@@ -18,6 +18,11 @@ set -e
 # shellcheck source=case-lib/lib.sh
 source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
 
+start_test
+# TODO: replace start_test with this:
```

Comment:
This TODO will be next PR?

Reply:
Yes immediately after this one. I want at least one run of daily tests between the two.